JWBestPracticeApps/FeedTableViewController/FeedTableViewController/Feed/FeedViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  FeedViewModel.swift
+//  FeedTableViewControllerV4
+//
+//  Created by Amitai Blickstein on 5/2/22.
+//
+
+import Foundation
+import JWPlayerKit
+
+/// All hard-coded and dynamic values required by the table view.
+/// Since there is only one d-source for a table view, this is a singleton.
+final class FeedViewModel {
+    // Singleton pattern
+    static let shared = FeedViewModel()
```

Comment:
As a developer I question why this class needs to be a singleton. When a class is a "model", that pattern should allow for multiple models to be loaded. Although outside of the scope for this BPA, someone who copies this code for their use might want to have 2 or more feeds loaded at the same time. 

Perhaps the table delegate should have an instance of this class instead.

Reply:
Refactored into a regular class.